lmdeploy/serve/openai/api_server.py

```diff
@@ -1035,6 +1049,12 @@ def serve(model_path: str,
         proxy_url (str): The proxy url to register the api_server.
         max_log_len (int): Max number of prompt characters or prompt tokens
             being printed in log. Default: Unlimited
+        concurrency_pressure: This refers to the ratio between the maximum
```

Comment:
Is this default value based on empirical experience?

Reply:
Yes, just an empirical number.